csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -1118,7 +1118,8 @@ dynamic_member_name(
 
 @qualifiable_expr = @member_access_expr
                   | @method_invocation_expr
-                  | @element_access_expr;
+                  | @element_access_expr
+                  | @delegate_invocation_expr;
```

Comment:
I guess we should add `QualifiableExpr` as a base class of `DelegateCall` too then? Even though it may look like an innocent change, simple changes to the type hierarchy have caused performance problems in the past, so we better run a dist comparison.